load-test/bin/undo-generate-pupil-pins.js

```diff
@@ -0,0 +1,36 @@
+const winston = require('winston')
+const fs = require('fs-extra')
+const sqlService = require('../../admin/services/data-access/sql.service')
+
+async function main () {
+  try {
+    winston.info('Deleting all pupil logon events...')
```

Comment:
as per slack chat - this can be a raw SQL script (doesnt leverage any benefit from JS), and its also a lot easier to run the raw sql as an elevated account that can delete from tables.

also - dont delete the jMeter output, this is useful